views/faq.erb

```diff
@@ -0,0 +1,16 @@
+<link rel="stylesheet" href="/styles/faq.css"/>
+
+<h1 class="title title-spacing">FAQ</h1>
+
+<ul>
+  <%= erb :faq_item %>
```

Comment:
Por como estas llamando a cada elemento, tendrías que tener un archivo para cada pregunta. Consultando con Julio me dió esta solución
```
<ul>
  <% faq_items.each do |faq| %>
      <%= erb :faq_item, question: faq.question, answer: faq.answer =>
  <% end %>
</ul>
```

Que haría un for que recorre un solo archivo con todas las preguntas